frontend/public/components/row-filter.jsx

```diff
@@ -102,7 +102,7 @@ class CheckBoxes_ extends React.Component {
     const all = _.map(this.props.items, 'id');
     const recognized = _.intersection(this.state.selected, all);
     if (!_.isEmpty(recognized)) {
-      this.props.applyFilter && this.props.applyFilter(recognized);
+      this.props.applyCustomFilter && this.props.applyCustomFilter(recognized);
```

Comment:
Not new, but you could use optional chaining

```suggestion
      this.props.applyCustomFilter?.(recognized);
```

Reply:
Done.